### DISCUSSION TITLE
Github Externship Programme

### DISCUSSION THREAD

**Discussion by salujajaskeerat** [github/india#13] [Students]
**_Can you please tell when does GitHub externship programme start?_**
Also, what is the eligibility criteria?

**Comment by arpangreat** [comment]
well we have to wait until the end of the event , after that we will know every details of the events

**Reply by sajid-sree12** [reply]
I am fresher to this externship platform, can anyone help me out to find application number in GitHub externship? I am approved to apply for GitHub externship but in my proposal letter I have to mention application number . anyone one help me out with this.

**Comment by GowthamGoush** [comment]
In fact, everyone is excited about it 🚀

**Comment by AnandChowdhary** [comment]
The eligibility seems to be:

> Third Year Students that are part of Higher-Ed institutions in India that have signed up for the GitHub Campus Program

More information is available at: https://github.com/github/india/blob/main/Students/EXTERNSHIPS.md

**Reply by varanasiroshan2001** [reply]
I am in second year and really interested can't I go in this🥺🥺

**Reply by kroy96** [reply]
Currently the program is open for only third-year engineering students of GCP schools.

Please check [here](https://github-externships.github.io/externship) for more details.

**Reply by mayank-cse** [reply]
Is there any program for 2nd year students
currently live or which may get live in future

If you know of any such program do share 
Thank You

**Reply by abhibhaw** [reply]
Yes just wait till you go in your 3rd year.

**Comment by AnandChowdhary** [comment]
How would a company go about offering GitHub Externships? We would love to sponsor students to work on our open-source projects, most notably @upptime.

**Reply by msupernaut** [reply]
@AnandChowdhary , Thank you for the offer to sponsor students. Can you drop a mail to india-externships[at]github.com so that the team can get in touch.

**Comment by anjali1102** [comment]
![Screenshot_2021-04-02-10-16-45-55](https://user-images.githubusercontent.com/56559378/113381569-ee78af80-939c-11eb-80b7-f9a4bc2ec0eb.png)

My institute is not listed in GITHUB CAMPUS PROGRAM PARTNERS SCHOOL. How can I add my institute here?

**Reply by punitjain03** [reply]
Currently the program is open for students only from GitHub Campus Partner schools. You can work with your faculty members and HOD to tell them about the campus program and motivate them to apply for the same.

The Campus program is totally free and with this your school will get a package of technical tools and access to many exclusive programs like GitHub Externship helping to grow the technical communities on your campus. More details and application process is available here [https://education.github.com/schools]

I would also encourage you to apply for many other offerings for students from GitHub Education.

**Reply by Tanujcbe** [reply]
Hi,
   I am interested in applying for Github Externship, But, even my college is not registered in GitHub Campus Partner schools. Can I apply by myself, or should I approach faculties? And what is the procedure for them to apply for GitHub Campus Partner schools?

**Comment by RASHI3004** [comment]
Is there any way to participate in the program if my institute is not listed in the GITHUB CAMPUS PROGRAM PARTNERS SCHOOL?

**Reply by vivekshotti** [reply]
Same doubt!

**Reply by punitjain03** [reply]
Currently the program is open for students only from GitHub Campus Partner schools. You can work with your faculty members and HOD to tell them about the campus program and motivate them to apply for the same.

The Campus program is totally free and with this your school will get a package of technical tools and access to many exclusive programs like GitHub Externship helping to grow the technical communities on your campus. More details and application process is available here [https://education.github.com/schools]

I would also encourage you to apply for many other offerings for students from GitHub Education.

**Reply by vivekshotti** [reply]
Okay buddy! Thanks a lot for informing 👍

**Comment by AshutoshDash1999** [comment]
Hey, I'm from a normal BSc college and my institute will not be interested in GitHub Campus Program. But I'm very much interested. How can I participate?

**Reply by adityamangal1** [reply]
Try to tell the advantages of this campus program to them.

**Reply by AshutoshDash1999** [reply]
It's not of any use, our college doesn't involve anything technical. It's like selling programming courses to bio students.

**Comment by kroy96** [comment]
> **_Can you please tell when does GitHub externship programme start?_**
> Also, what is the eligibility criteria?

Please visit [GitHub Externship website](https://github-externships.github.io/externship) for complete details regarding the program.

**Comment by leoshrey** [comment]
Hey, Extremely excited to participate in this program. As my school is listed on the **Partner Schools list**, I would like to ask what's the process after I find my school in the list:
- Do I only have to wait for the organization application period to end ? (i.e. 21 April 2021) then move for applying as a student 
OR
- I can contribute to the organization's projects even before the students application starts, if somehow I can find what organization will be participating. 


It would be great to see a reply.

**Reply by kroy96** [reply]
To answer your queries:

You can start contributing to the participating organization's projects even before the student application start, and regarding your question of where do you view the participating organization's and their projects, those will be listed on the following website in the organization section soon https://github-externships.github.io/externship/organizations.html

**Reply by leoshrey** [reply]
Thank you for your response, Excited to see the updation.

**Comment by leoshrey** [comment]
Thank you for your response, Excited to see the updation.

**Reply by leoshrey** [reply]
Hey !, Do we need to apply for campus program before applying for externship even if my campus is listed on campus partners list ?

**Reply by vivekshotti** [reply]
Yes, buddy. If your educational institute isn't a partner with Github, you will have to convince your college/university to enroll and then move on with the externship program process.

**Reply by leoshrey** [reply]
My school is listed in the campus list. Then I think I don't have to apply for campus program.

**Reply by leoshrey** [reply]
@Vivek-Hotti Hey, I noticed some organisations are listed on the Org. page. Can I know if these organisations are ready to be applied to or these are just for website experiments as the links provided there are not redirecting to any page. Also, a very brief description of the org. is provided.

**Comment by ishan-sharma-3004** [comment]
What all do we have to submit in the proposal can anyone provide some information on this part please??

**Reply by kroy96** [reply]
Hi @paramagneticnaga we have got a Slack workspace with Mentors from all the organization, please do join it for your project related queries as well you will find all the details on the Organization project page.
Slack Workspace - https://join.slack.com/t/githubexternship/shared_invite/zt-pa2a0lja-slJzOZuRmxjKOnUOk12rVQ

**Comment by Tanujcbe** [comment]
Hi,
   I am interested in applying for Github Externship, But, even my college is not registered in GitHub Campus Partner schools. Can I apply by myself, or should I approach faculties? And what is the procedure for them to apply for GitHub Campus Partner schools?

**Reply by GowthamGoush** [reply]
The program is only open to students from GitHub Campus partner schools, and unfortunately, the last date to onboard schools making their students eligible for the Externship program is over. However, you can still register your school for GitHub Campus partner to make it eligible for the next batch. All the best!

**Comment by sab30-webdev** [comment]
Is github winter externship true ? What is its timeline ?

**Comment by roshniLingam** [comment]
Is there any selection processes for the GitHub externship? If yes, please tell me about it.

**Comment by har790** [comment]
what is the application deadline??
and  how many days does it take for a clg to get in the list after applying for GitHub campus program

**Comment by Anjali0739** [comment]
Where do I submit my proposals for GitHub externship?

**Reply by sajid-sree12** [reply]
You can submit proposal at the time of applying. After clicking 'apply' button you will get 3rd option as to upload proposal pdf file, proposal pdf should be less than 15MB. hurry up today is the last date for submission.

**Comment by akshitadixit** [comment]
Do the applications open just once a year?

**Reply by sajid-sree12** [reply]
Externship cohort repeats twice a year.

**Reply by akshitadixit** [reply]
@sajid-sree12 thanks for the reply, may I know approximate dates/months for the same?

**Reply by sajid-sree12** [reply]
I don't know exactly, may be roughly 6 months.

**Comment by BhaswatiRoy** [comment]
my college registered for github extrenship eligible schools. so will we get a mail regarding whether our college is registered or not or is there any list from where we can check whether our college got enrolled in the same or not?

**Reply by akshitadixit** [reply]
The point of contact i.e. the facult-in-charge who registered for GitHub Campus Education would receive a confirmation, you can confirm from the concerned faculty.
